### PR TITLE
feat(service-portal): login and layout

### PR DESCRIPTION
**Pull request by thordurhhh**
## What

New design layout for launch of minar-sidur.

## Screenshots / Gifs

https://www.figma.com/proto/sMnUuOHOGuill3O3hxEICG/H%C3%B6nnun-m%C3%ADnar-s%C3%AD%C3%B0ur?page-id=7259%3A0&node-id=7307%3A1207&viewport=297%2C873%2C0.16715258359909058&scaling=min-zoom

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by Valur — APPROVED** [review]
LGTM

File: apps/web/components/Header/Header.tsx [hunk 1]
```diff
@@ -89,7 +89,10 @@ export const Header: FC<HeaderProps> = ({
                     )}
                     <Hidden below="lg">
                       <Box marginLeft={marginLeft}>
-                        <Link href="//minarsidur.island.is/" skipTab>
+                        <Link
+                          href={english ? '/en/login' : '/innskraning'}
```
Comment (stefanhar) [inline apps/web/components/Header/Header.tsx:93]: Consider adding this url to the link resolver to keep internal link resolution consistent

File: apps/web/components/Menu/Menu.tsx [hunk 2]
```diff
@@ -94,7 +94,10 @@ export const Menu = ({
       }}
       renderMyPagesButton={(button) => {
         return (
-          <Link href="//minarsidur.island.is/" skipTab>
+          <Link
+            href={activeLocale === 'en' ? '/en/login' : '/innskraning'}
```
Comment (stefanhar) [inline apps/web/components/Menu/Menu.tsx:98]: Consider adding this url to the link resolver to keep internal link resolution consistent

**Review by stefanhar — CHANGES_REQUESTED** [review]

File: apps/web/components/Header/Header.tsx (hunk 1 above)
Comment (thordurhhh) [reply apps/web/components/Header/Header.tsx:93]: Yes! Makes sense.

File: yarn.lock [hunk 3]
```diff
@@ -21537,7 +21537,7 @@ postcss-value-parser@^4.0.0, postcss-value-parser@^4.0.2, postcss-value-parser@^
   resolved "https://registry.yarnpkg.com/postcss-value-parser/-/postcss-value-parser-4.1.0.tgz#443f6a20ced6481a2bda4fa8532a6e55d789a2cb"
   integrity sha512-97DXOFbQJhk71ne5/Mt6cOu6yxsSfM0QGQyl0L25Gca4yGWEGJaig7l7gbCX623VqTBNGLRLaVUCnNkcedlRSQ==
 
-postcss@6.0.1, postcss@7.0.21, postcss@7.0.32, postcss@^6.0.1, postcss@^7.0.1, postcss@^7.0.11, postcss@^7.0.14, postcss@^7.0.17, postcss@^7.0.18, postcss@^7.0.23, postcss@^7.0.26, postcss@^7.0.27, postcss@^7.0.32, postcss@^7.0.5, postcss@^7.0.6, "postcss@npm:@island.is/postcss@107.0.37":
+postcss@6.0.1, postcss@7.0.21, postcss@7.0.32, postcss@^6.0.1, postcss@^7.0.0, postcss@^7.0.1, postcss@^7.0.11, postcss@^7.0.14, postcss@^7.0.17, postcss@^7.0.18, postcss@^7.0.23, postcss@^7.0.26, postcss@^7.0.27, postcss@^7.0.32, postcss@^7.0.5, postcss@^7.0.6, "postcss@npm:@island.is/postcss@107.0.37":
```
Comment (sindrig) [inline yarn.lock:21540]: not sure why you have changes here?

Comment (thordurhhh) [reply yarn.lock:21540]: I just pulled the most recent main and did a `yarn install`

Seems to be reverting this change from two hours ago: https://github.com/island-is/island.is/commit/aef0e95adb3177bcb70359dd4f75cb956214bccd#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR21446

File: apps/web/components/Menu/Menu.tsx (hunk 2 above)
Comment (thordurhhh) [reply apps/web/components/Menu/Menu.tsx]: This has been changed ✅

**Comment by thordurhhh** [comment]
Any chance of a re-review?
Thanks.

File: yarn.lock (hunk 3 above)
Comment (thordurhhh) [reply yarn.lock]: Reverted.

File: apps/web/components/Header/Header.tsx (hunk 1 above)
Comment (thordurhhh) [reply apps/web/components/Header/Header.tsx]: @stefanhar - this has been fixed. Any chance of a re-review when you got the time 🙏  😃

**Review by stefanhar — APPROVED** [review]
LGTM

**Review by eirikurn — APPROVED** [review]

**Pull request merged**